configs/projects/_shared-agent-components.rb

```diff
@@ -51,6 +51,7 @@
 proj.component 'rubygem-locale'
 proj.component 'rubygem-gettext'
 proj.component 'rubygem-fast_gettext'
+proj.component 'rubygem-scanf'
```

Comment:
This should be only for agent-runtime-next.rb which I'm adding in https://github.com/puppetlabs/puppet-runtime/pull/360.

So you can just add the component here then I'll rebase my PR.